TESTING.md

```diff
@@ -35,33 +35,87 @@ The unit tests can be executed in one of two ways:
+
+
+### Limitations in our code coverage reporting
+Our code coverage implementation has some known limitations:
+- [Variability](https://github.com/nasa/openmct/issues/5811)
+- [Accuracy](https://github.com/nasa/openmct/issues/7015)
+- [Vue instrumentation gaps](https://github.com/nasa/openmct/issues/4973)
+
+## Troubleshooting CI
+The following is an evolving guide to trouble CI and PR issues.
+
+### Github Checks failing
+There are a few reasons that your Github PR could be failing beyond simple failed tests.
+* Required Checks. We're leveraging required checks in Github so that we can quickly and precisely control what becomes and informational failure vs a hard requirement. The only way to determine the difference between a required vs information check is check for the `(Required)` emblem next to the step details in Github Checks.
+* Not all required checks are run per commit. You may need to manually trigger addition github checks with a `pr:<label>` label added to your PR.
+
+### Flaky tests
+There are two ways to know if a test on your branch is historically flaky: 
+1. `deploysentinel`'s PR comment bot to give an accurate and historical view of e2e flakiness. Check your PR for a view of the test failures and flakes (with link to the failing test). Note: only a 7 day window of flake is available.
+2. (CircleCI's test insights feature)[https://circleci.com/blog/introducing-test-insights-with-flaky-test-detection/] collects historical data about the individual test results for both unit and e2e tests. Note: only a 14 day window of flake is available.
+
+### Local=Pass and CI=Fail
+Although rare, it is possible that your test can pass locally but fail in CI.
 
 #### Busting Cache
 In certain circumstances, the CircleCI cache can become stale. In order to bust the cache, we've implemented a runtime boolean parameter in Circle CI creatively name BUST_CACHE. To execute:
 1. Navigate to the branch in Circle CI believed to have stale cache.
 1. Click on the 'Trigger Pipeline' button.
 1. Add Parameter -> Parameter Type = boolean , Name = BUST_CACHE ,Value = true
-1. Click 'Trigger Run'
-
-## Code Coverage
+1. Click 'Trigger Pipeline'
 
-* 100% statement coverage is achievable and desirable.
+#### Run tests in the same container
 
-Codecov.io will combine each of the above commands with [Codecov.io Flags](https://docs.codecov.com/docs/flags). Effectively, this allows us to combine multiple reports which are run at various stages of our CI Pipeline or run as part of a parallel process.
+In extreme cases, tests can fail due to the constraints of running within a container. To execute tests in exactly the same way as run in CircleCI. To date, we've not had github actions fail differently than local execution.
```

Comment:
`To execute tests in exactly the same way as run in CircleCI`

this sentence seems out of place?